src/sorcha/ephemeris/simulation_constants.py

```diff
@@ -1,5 +1,7 @@
 import numpy as np
 
+GMSUN = 2.9591220828559115e-04
+GMTOTAL = 2.963092748799319386e-04
```

Comment:
Can we also get GMtotal from the auxiliary ephemeris files?

Reply:
GMTOTAL = sum(sorted([ephem.get_particle(i, 0).m for i in range(27)]))

Reply:
I inserted a calculation of gm_total into create_assist_ephemeris, but I did not alter the returned values to include gm_total yet.

Reply:
Can you remove GMTOTAL and GMSUN I worry that we'll have these constants sitting around and a later date someone will try to use these and not the ones from simulation_setup.py. I think one those are gone then it's good to go for a merge (once we sort the unit tests)